src/query.rs

```diff
@@ -40,20 +40,20 @@ impl ActualSemverUpdate {
 /// A query that can be executed on a pair of rustdoc output files,
 /// returning instances of a particular kind of semver violation.
 #[derive(Debug, Clone, Serialize, Deserialize)]
-pub(crate) struct SemverQuery {
-    pub(crate) id: String,
+pub struct SemverQuery {
```

Comment:
```suggestion
#[non_exhaustive]
pub struct SemverQuery {
```
We may want to add new fields in the future here.

Reply:
done in https://github.com/obi1kenobi/cargo-semver-checks/pull/314/commits/368ee5b0c0e866e8dedc00d079ff4ac63cd764e1